Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -13,7 +13,10 @@ import Foundation
 class DashboardViewModel: ObservableObject {
     let authenticationViewModel: AuthenticationViewModel
     
+    @Published var showAddView = false
```

Comment:
Using this boolean is one way to do this, but another is:
```swift
NavigationLink {
    AddMaintenanceView() { event in
        Task {
            await viewModel.addEvent(event)
        }
    }
} label: {
    Image(systemName: "plus")
}
```

Reply:
This makes sense and simplifies things by allowing us to delete the boolean! I was under the impression that since we were using a NavigationStack I had to use NavigationPath. Guess not! Thanks!

Reply:
@achi113s Turns out you do need to use this boolean! haha I couldn't figure out how to pop the view after the event was added

Reply:
hahaha its always the smallest features that cause the most headaches